README.md

```diff
@@ -25,6 +31,14 @@ This project uses the Stencil testing framework for unit and end-to-end (E2E) te
 
 Below are commands used for various testing and development scenarios.
 
+## Build
+
+Before building the project, make sure to set the correct environment using the `NODE_ENV` variable. The build scripts for different environments are:
+
+- `yarn run build:dev` for development environment.
+- `yarn run build:staging` for staging environment.
```

Comment:
It looks like there is not a `.env.staging` - can we add one? `IFRAME_ORIGIN=https://js.justifi-staging.com/v2`